opentelemetry-metrics-exporters/src/main/java/com/cumulocity/exporters/platform/PlatformMetricExporter.java

```diff
@@ -0,0 +1,107 @@
+        double value = 0;
+        if (metric.getType().equals(LONG_SUM)) {
+            value = metric.getLongSumData().getPoints().stream().findFirst().get().getValue();
+        } else if (metric.getType().equals(DOUBLE_SUM)) {
+            value = metric.getDoubleSumData().getPoints().stream().findFirst().get().getValue();
+        } else {
+            value = 0;
+        }
+        log.info("Sending metric for tenant {} named {} whose value is {} and whose type is {}", restConnector.getPlatformParameters().getTenantId(), name, value, metric.getType().name());
+        MicroserviceBillingRepresentation representation = new MicroserviceBillingRepresentation();
+        representation.setDateTime(DateTime.now());
+        representation.set(value, name);
+        restConnector.post("/application/currentApplication/billing",
+                APPLICATION_JSON_TYPE, representation);
+    }
+
+    private void exportGaugeMetrics(MetricData metric) {
+        Collection<DoublePointData> points = metric.getDoubleGaugeData().getPoints();
+        Map<DoublePointData, DateTime> pointDataDateTimeMap = new HashMap<>();
+        for(DoublePointData pointData: points) {
+            Attributes attributes = pointData.getAttributes();
+            List<Object> values = attributes.asMap().values().stream().collect(Collectors.toList());
+            DateTime dateTime = DateTime.parse((String) values.get(0));
+            pointDataDateTimeMap.put(pointData, dateTime);
+        }
+        DoublePointData pointToExport = Collections.max(pointDataDateTimeMap.entrySet(), Map.Entry.comparingByValue()).getKey();
+        String name = metric.getName();
+        double value = pointToExport.getValue();
+        MicroserviceBillingRepresentation representation = new MicroserviceBillingRepresentation();
+        representation.setDateTime(DateTime.now());
+        representation.set(value, name);
+        restConnector.post("/application/currentApplication/billing",
+                APPLICATION_JSON_TYPE, representation);
+    }
+
+    @Override
+    public CompletableResultCode export(Collection<MetricData> collection) {
+        List<MetricData> gaugeMetrics = collection.stream().filter(metric -> metric.getType().equals(DOUBLE_GAUGE)).collect(Collectors.toList());
+        List<MetricData> otherMetrics = collection.stream().filter(metric -> !metric.getType().equals(DOUBLE_GAUGE)).collect(Collectors.toList());
+        int gaugePointSize = 0;
+        for(int gaugeIndex = 0; gaugeIndex < gaugeMetrics.size(); gaugeIndex++) {
+            gaugePointSize += gaugeMetrics.get(gaugeIndex).getDoubleGaugeData().getPoints().size();
+        }
+        for(MetricData metric : otherMetrics) {
+            exportNonGaugeMetrics(metric);
+        }
+        for(MetricData metric: gaugeMetrics) {
+            exportGaugeMetrics(metric);
+        }
+        applicationEventPublisher.publishEvent(new ExportCompletedEvent("exportCompletedEvent", gaugePointSize));
```

Comment:
Limitation with Gauge Metric Instrumentation: The callbacks registered on instrumented the metric arent closed on export. Hence we need to publish an event, which when received by the controller/service, instrumenting the metrics, closes the list of gauges stored on instrumentation